value/value.md

```diff
@@ -231,87 +225,183 @@ the player to your app, and for which nothing already exists, add your own.
 
 ### Payment calculation
 
-The interval payment calculation is:
+When all combined, together the recipients will receive
+```
+(Batch payout) = (Interval count) * (Unit amount)
+```
+
+`Unit amount` refers to one of the following
+- either the amount of a one-time payment (with `Interval count` usually being 1)
+- or the amount being streamed per minute (with `Interval count` referring to the interval in minutes at which the payouts are made)
+
+How `Batch payout` is divided between the recipients depends on their `split` and `fee` values.
 
-    (Number of shares / Share total) * Interval payout * Interval count
+For `fee=false` recipients $X_1, X_2, \ldots, X_m$ with splits $x_1, x_2, \ldots, x_m$ and `fee=true` recipients $Y_1, Y_2, \ldots, Y_n$ with splits $y_1, y_2, \ldots, y_n$:
```

Comment:
Make X first since fee payments are first?
```suggestion
For `fee=true` recipients $X_1, X_2, \ldots, X_m$ with splits $x_1, x_2, \ldots, x_m$ and `fee=false` recipients $Y_1, Y_2, \ldots, Y_n$ with splits $y_1, y_2, \ldots, y_n$:
```